src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -215,6 +216,10 @@ public function load(array $configs, ContainerBuilder $container)
             if (!class_exists(BaseYamlLintCommand::class)) {
                 $container->removeDefinition('console.command.yaml_lint');
             }
+
+            if (!class_exists(MailerDebugCommand::class)) {
+                $container->removeDefinition('console.command.mailer_debug');
```

Comment:
Allow to remain BC if the `symfony/mailer` component is not installed.

Maybe there is a better way ?